indexd/index/blueprint.py

```diff
@@ -228,16 +245,21 @@ def post_index_record():
     except jsonschema.ValidationError as err:
         raise UserError(err)
 
+    urls = flask.request.json['urls']
+    urls_metadata = flask.request.json['urls_metadata']
+    # Should be the same on both ends.
+    if sorted(urls) != sorted(urls_metadata.keys()):
```

Comment:
Do you think it'd be good enough to have a database constraint on `type IS NOT NULL`? I wonder if doing that will have some sort of affect on the data migration.

Reply:
I think we do have a type for every URL in the GDC, but I'm afraid to find out we missed some edge case... so probably better not to set the constraint, I guess.

My comment was actually about making `urls_metadata` a required field when posting a new record. I commented here because I noticed the use of brackets to access `urls_metadata`, but I probably should've put the comment in `schema.py` instead. Anyway, I don't think we actually need to change anything.